python/setup.py

```diff
@@ -168,6 +170,9 @@
 if not libcuml_path:
     libcuml_path = '../cpp/build/'
 
+# for now, do not build umap for cuda-11
+cython_exc_list.append('cuml/manifold/umap.pyx')
```

Comment:
Similar to Dante's comments above, we should be able to add this back in now

Reply:
Done